views/tx.tmpl

```diff
@@ -268,8 +268,12 @@
                   <td class="text-left py-1 text-secondary" data-target="tx.spendingTx">
                     {{$spendingTx := index .SpendingTxns 0}}
                     {{if $spendingTx.Hash}}
-                      <a href="/tx/{{$spendingTx.Hash}}"
-                      >{{if eq .TicketInfo.PoolStatus "missed"}}revocation{{else}}vote{{end}}
+                     <a href="/tx/{{$spendingTx.Hash}}"
+                      >{{if or (eq .TicketInfo.PoolStatus "missed") (eq .TicketInfo.PoolStatus "expired")}}
+                      revocation
+                      {{else}}
+                      vote
+                      {{end}}
```

Comment:
Note that this ends up as:

```html
                     <a href="/tx/732ab27c34739acbc5e4e652580b092e044578a6fac7d85f860a96c99ea01a99">
                      revocation
                      
                      </a>
```

or

```html
                      <a href="/tx/ef647c0ff841fb03642f4f91de43e20a927e0e39b11d5ae2ff66b6698a22fd83">
                      vote
                      
                      </a>
```

Note that the blank line is actually 22 spaces.

Would you please eat whitespace like (and adjust the indentation of the `<a>`) tags:

```suggestion
                    {{if $spendingTx.Hash}}
                     <a href="/tx/{{$spendingTx.Hash}}"
                      >{{if or (eq .TicketInfo.PoolStatus "missed") (eq .TicketInfo.PoolStatus "expired") -}}
                      revocation
                      {{- else -}}
                      vote
                      {{- end -}}
                    </a>
```

Reply:
Changes completed.